src/pages/EMPReplicantTargets.page

```diff
@@ -0,0 +1,18 @@
+<apex:page standardController="EMP_Organization_Member__c" extensions="EMPReplicantProfile" sidebar="false" language="es">
+<apex:form >
+    <apex:pageBlock >
+      <apex:pageBlockSection title="Metas">
+        <apex:pageBlockTable value="{!targetDiagnostic}" var="fTD">
+            <apex:column headerValue="Fecha Visita">
+            <apex:outputLink target="_blank" value="/{!fTD.Id}">{!fTD.Visit_Date__c}</apex:outputLink>
```

Comment:
@KayLuke do you know how can I format the date in this tag (field fTD.Visit_Date__c)? Thanks

<apex:outputLink target="_blank" value="/{!fTD.Id}"> {!fTD.Visit_Date__c} </apex:outputLink>